segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/ContainerEventProcessorTests.java

```diff
@@ -189,24 +194,11 @@ public static void testFaultyHandler(ContainerEventProcessor eventProcessorServi
         AssertExtensions.assertEventuallyEquals(true, () -> retries.get() == 10, 10000);
     }
 
-    /**
-     * Check that multiple EventProcessors can work in parallel under the same {@link ContainerEventProcessor}, even in if
-     * some of them are faulty.
-     *
-     * @throws Exception
-     */
-    @Test(timeout = TIMEOUT_SUITE_MILLIS)
```

Comment:
I removed this intentionally because with the change in refreshing segment, mocking this test is quite difficult. On the other hand, this very exact logic is properly tested with real Segments in `StreamSegmentContainerTests` (in fact, we have several tests that are testing the same logic and the only change is to use a `MockSegment` (`ContainerEventProcessorTests`) or a real one (`StreamSegmentContainerTests`)). Therefore, due to the complexity of instrumenting the mock to do what I need and the fact that the logic is actually validated already with a real Segment, I just decided to remove it (it is providing no extra validation and make it to work is cumbersome).